stream-compositions/events/pom.xml

```diff
@@ -15,6 +15,12 @@
 
     <name>Stream :: Compositions :: Events</name>
 
+    <properties>
+        <!-- TODO: This module shouldn't be inheriting from SSDK Starter Parent, disabling Jib plugin as workaround -->
```

Comment:
@piotr-backbase we need to address this anytime

Reply:
why it should not ?

Reply:
@piotr-backbase  The SSDK starters are meant to be used as parents to services only and not libraries, they come with all the plugins configured in a way to be used to ship a runnable artefact (docker images our executable jars) which is not the case for libraries, hence why we had to manually disable the respective plugins to avoid crashing the deploy phase of the maven execution.

ref: https://community.backbase.com/documentation/ServiceSDK/latest/service_sdk_starters

Reply:
ok, makes sense.